rwandaemr-installer/src/main/resources/scripts/create-service-user.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+## load variables from the vars.txt file
+source vars.txt
+
+home_dir=${parent_dir}
+
+if [ "$env_type" == "demo" -o "$env_type" == "prod" -o "$env_type" == "test" ]; then
+  echo "Checking if user doesn't exist"
+  if [ $(id -u) -eq 0 ]; then
+	egrep "^$username" /etc/passwd >/dev/null
+	if [ $? -eq 0 ]; then
+		echo "$username exists! Skipping!"
+		exit 1
+	else
+		useradd -m ${username} -d /opt/${home_dir} -s /bin/bash
+		[ $? -eq 0 ] && echo "User has been added to system!" || echo "Failed to add a user!"
```

Comment:
I feel the opposite here.  Shouldn't we fail hard if there is an error creating a user?  We only get here if the user doesn't exist, so we are expecting a user to get created here.  If there is a problem doing so, we want the process to stop and to indicate that there was an unexpected error here.  Put another way - this is an unexpected error we want to expose.  The above (if a user already exists) is an expected condition we want to allow.